### DISCUSSION TITLE
增加一个订阅更新通知的功能怎么样。。

### DISCUSSION THREAD

**Discussion by Akoasmn** [spencerwooo/onedrive-vercel-index#654] [Ideas]
先感谢阅读
我觉得增加一个更新通知的功能会很棒，
比如通过邮箱通知，用户可以选择订阅某一特定文件夹。当文件夹有内容更新时自动使用邮箱通知。
毕竟大家不会反复打开网站查看自己喜欢的东西有没有更新吧。。。

**Comment by myl7** [comment]
OneDrive 没有提供相应的检查文件变动的 API，所以很难很难实现，也超出了此项目原有的目的
